packages/lab/src/pill/Pill.css

```diff
@@ -0,0 +1,201 @@
+.uitk-density-high {
+  --pill-icon-offset: 1px;
+}
+
+.uitk-density-medium {
+  --pill-icon-offset: 2px;
+}
+
+.uitk-density-low {
+  --pill-icon-offset: 3px;
+}
+
+.uitk-density-touch {
+  --pill-icon-offset: 4px;
+}
+
+/* Styles applied to the root element */
+.uitkPill {
+  --pill-background: var(--uitkPill-background, var(--uitk-taggable-background));
+  --pill-background-active: var(--uitkPill-background-active, var(--uitk-taggable-background-active));
+  --pill-background-disabled: var(--uitkPill-background-disabled, var(--uitk-taggable-background-disabled));
+  --pill-background-hover: var(--uitkPill-background-hover, var(--uitk-taggable-background));
+  --pill-checkbox-size: var(--uitkPill-checkbox-size, calc(var(--uitk-size-pill) - 2px));
+  --pill-icon-color: var(--uitkPill-icon-color, var(--uitk-taggable-foreground));
+  --pill-icon-color-active: var(--uitkPill-icon-color-active, var(--uitk-taggable-foreground-active));
+  --pill-icon-color-hover: var(--uitkPill-icon-color-hover, var(--uitk-taggable-foreground-hover));
+  --pill-text-color: var(--uitkPill-text-color, var(--uitk-taggable-foreground));
+  --pill-text-color-active: var(--uitkPill-text-color-active, var(--uitk-taggable-foreground-active));
+  --pill-text-color-hover: var(--uitkPill-text-color-hover, var(--uitk-taggable-foreground-hover));
+}
+
+/* Button Component API - reset to default state so they don't interfere with Pill styling */
+.uitkPill {
+  --uitkButton-background-active: var(--pill-background);
+  --uitkButton-textColor-active: var(--pill-text-color);
+  --uitkButton-background-hover: var(--pill-background);
+  --uitkButton-textColor-hover: var(--pill-text-color);
```

Comment:
I've lost track of what we decided on textColor vs text-color. We seem to have both